ocs_ci/utility/kms.py

```diff
@@ -89,7 +89,9 @@ def __init__(self):
         self.vault_path_token = None
         self.vault_policy_name = None
         self.vault_kube_auth_path = "kubernetes"
+        self.vault_kube_auth_role = constants.VAULT_KUBERNETES_AUTH_ROLE
```

Comment:
isn't this supposed to be conditional ? I mean we will enable this only if auth method is kubernetes ?

Reply:
Yes, the `vault_path_token` `vault_kube_auth_path` and `vault_kube_auth_role` are parameters that will depend on the type of auth method being used. This is handled in the functions below, so that they don't affect each other